nemo/collections/tts/torch/data.py

```diff
@@ -277,6 +281,9 @@ def __init__(
         self.sup_data_types_set = set(self.sup_data_types)
 
         for data_type in self.sup_data_types:
+            if data_type not in VALID_SUPPLEMENTARY_DATA_TYPES:
+                raise NotImplementedError(f"Current implementation doesn't support {data_type} type.")
+
```

Comment:
Please remove this exception. See details in this PR (https://github.com/NVIDIA/NeMo/pull/4297).

Reply:
Done.